shardingsphere-kernel/shardingsphere-sql-federation/shardingsphere-sql-federation-optimizer/pom.xml

```diff
@@ -86,4 +86,25 @@
             <artifactId>calcite-core</artifactId>
         </dependency>
     </dependencies>
+    <build>
```

Comment:
Why add antlr plugin in shardingsphere-sql-federation-optimizer?

Reply:
Because the ANTLR is used to unserialize the filter condition.